README.adoc

```diff
@@ -12,60 +12,40 @@ Refer to the link:https://docs.cloudbees.com/docs/cloudbees-platform/latest/work
 .Input details
 |===
 
-| Input name
-| Data type
-| Required?
-| Description
-
-| `approvers`
-| String
-|No
+.^| Input name
+.^| Data type
+.^| Required?
+.^| Description
+
+.^| `approvers`
+.^| String
+.^|No
 | A list of users whose participation in the workflow approval process is requested.  If left blank, all eligible users will be notified.
 
-| `timeout-minutes`
-| Integer
-| No
-| The amount of time approvers have to respond to the approval request.  The default value is `4320` minutes.
-
-| `delegates`
-|String
-| Yes
+.^| `delegates`
+.^|String
+.^| Yes
 | The value must be the path to the Manual approval custom job yaml file: `cloudbees-io/manual-approval/custom-job.yml@v1`
 
-| `disallowLaunchByUser`
-|String
-| No
-| When set to true, it prevents the user who started the workflow from participating in the approval.  Default value is `false`.
-
-| `notifyAllEligibleUsers`
```

Comment:
We can remove this field `notifyAllEligibleUsers` from the Usage section too.

Reply:
done